lib/ssh_data/encoding.rb

```diff
@@ -313,6 +326,21 @@ def decode_string_public_key(raw, offset=0, algo=nil)
       [key, str_read]
     end
 
+    def decode_openssh_signature(raw, offset=0)
+      total_read = 0
+
+      magic = raw.byteslice(total_read, OPENSSH_SIGNATURE_MAGIC.bytesize)
```

Comment:
Should there be some offset included here?

Reply:
Yep, started respecting offset and added a test, even though the higher level signature decoder will never pass in an offset.